frontend/lib/src/hocs/withMapboxToken/MapboxToken.ts

```diff
@@ -55,13 +49,7 @@ export class MapboxToken {
       if (userMapboxToken !== "") {
         MapboxToken.token = userMapboxToken
       } else {
-        // TODO: Replace this with the block below after October 1st 2020.
         MapboxToken.token = await this.fetchToken(TOKENS_URL, "mapbox")
-        // if (this.isRunningLocal() && SessionInfo.isHello) {
-        //   MapboxToken.token = await this.fetchToken(TOKENS_URL, "mapbox-localhost")
-        // } else {
-        //   throw new MapboxTokenNotProvidedError("No Mapbox token provided")
-        // }
```

Comment:
We plan to maintain current behavior - not going to transition to forcing users to use their own Mapbox token.
Slack [confirmation](https://snowflake.slack.com/archives/C039V2A2USJ/p1689353931500289)